geomstats/visualization/beta_distributions.py

```diff
@@ -0,0 +1,426 @@
+"""Visualization for Geometric Statistics."""
```

Comment:
Rename `beta_distributions.py` --> `beta.py`
Change docstring: Visualization for Beta distributions.